inc/switchtec/switchtec.h

```diff
@@ -391,11 +394,12 @@ void switchtec_perror(const char *str);
 int switchtec_log_to_file(struct switchtec_dev *dev,
 			  enum switchtec_log_type type,
 			  int fd,
-			  FILE *log_def_file);
+			  FILE *log_def_file,
+			  struct switchtec_log_file_info *info);
```

Comment:
This seems really odd given that the info struct is just being used to return warning flags and the _version fields aren't even set by this call. Why not just return some positive flags or something on success with warnings?

Reply:
Yes I forgot to set the version fields in the previous patch. Update now.